flask/notejam/models.py

```diff
@@ -6,11 +6,12 @@
 
 from notejam import db
 
+PASSWORD_LEN = len(generate_password_hash('foo'))
 
 class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(120), unique=True)
-    password = db.Column(db.String(36))
+    password = db.Column(db.String(PASSWORD_LEN))
```

Comment:
And that day we can update the column :) It's just a learning app, let's keep it as simple as possible.